### PR TITLE
Slider 'enableShadow' prop and ColorPicker - minor UI changes

### PR DESCRIPTION
**Pull request by Inbal-Tish**
## Description
Slider - add 'enableShadow' prop 
GradientSlider - display color only changes on hue change
ColorPicker - minor UI changes
Slider - bug fix - invoke onChange only when user move slider

## Changelog
Slider - add 'enableShadow' prop 
GradientSlider - display color only changes on hue change
ColorPicker - minor UI changes
Slider - bug fix - invoke onChange only when user move slider

## Additional info
MADS-3847

### CODE REVIEW

File: src/components/colorPicker/ColorPickerPreview.tsx [hunk 1]
```diff
@@ -89,7 +89,11 @@ const styles = StyleSheet.create({
   preview: {
     height: 200,
     alignItems: 'center',
-    justifyContent: 'center'
+    justifyContent: 'center',
+    borderBottomWidth: 0.5,
+    borderBottomColor: Colors.$outlineDisabled,
+    borderTopWidth: 0.5,
```
Comment (ethanshar) [inline src/components/colorPicker/ColorPickerPreview.tsx:95]: Consider using `StyleSheet.hairlineWidth` for the thin border..

File: src/components/slider/GradientSlider.tsx [hunk 2]
```diff
@@ -83,9 +83,9 @@ const GradientSlider = (props: Props) => {
     ...others
   } = props;
 
-  const [initialColor] = useState(Colors.getHSL(propsColors));
+  const initialColor = useRef(Colors.getHSL(propsColors));
   const [color, setColor] = useState(Colors.getHSL(propsColors));
-
+
```
Comment (ethanshar) [inline src/components/slider/GradientSlider.tsx:88]: Remove extra space

File: src/components/slider/GradientSlider.tsx [hunk 3]
```diff
@@ -94,25 +94,31 @@ const GradientSlider = (props: Props) => {
     return color || sliderContext.value;
   }, [color, sliderContext.value]);
 
-  const renderDefaultGradient = useCallback(() => {
+  const getHueColor = useCallback(() => {
```
Comment (ethanshar) [inline src/components/slider/GradientSlider.tsx:97]: It feels like this should be a memoize value more than a callback.. WDYT?

**Review by ethanshar — COMMENTED** [review]
Wrote small comments. 
Anyway approved. 

Btw, I pushed a small change in the screen name

**Review by ethanshar — APPROVED** [review]

File: src/components/slider/GradientSlider.tsx (hunk 2 above)
Comment (Inbal-Tish) [reply src/components/slider/GradientSlider.tsx:88]: This is not extra, I separated the consts and the useEffect

File: src/components/colorPicker/ColorPickerPreview.tsx (hunk 1 above)
Comment (Inbal-Tish) [reply src/components/colorPicker/ColorPickerPreview.tsx:95]: Nice!

File: src/components/slider/GradientSlider.tsx (hunk 3 above)
Comment (Inbal-Tish) [reply src/components/slider/GradientSlider.tsx:97]: Yes. Good idea

**Pull request merged**